tests/test_message_sync.py

```diff
@@ -223,6 +223,7 @@ def test_ReplySync_exception(mocker):
     mocker.patch("sdclientapi.sdlocalobjects.Reply", mocker.MagicMock(side_effect=Exception()))
 
     rs = ReplySync(api, home, is_qubes)
+    mocker.patch.object(rs, 'session_maker')
```

Comment:
What error prompted this? I think it would be better to let a real session get created to ensure that we catch any bugs with the session / home dir here.

Reply:
Resolving now that this is outdated after removing session code from gui.